lte/gateway/release/build-ovs.sh

```diff
@@ -40,14 +40,15 @@
 
 set -e
 WORK_DIR=~/build-ovs
-OVS_VERSION="v2.8.1"
-OVS_VERSION_SHORT="2.8.1"
+OVS_VERSION="v2.8.9"
+OVS_VERSION_SHORT="2.8.9"
 MAGMA_ROOT="../../../"
-GTP_PATCH_PATH="${MAGMA_ROOT}/third_party/gtp_ovs"
+GTP_PATCH_PATH="${MAGMA_ROOT}third_party/gtp_ovs/kenrel-4.9"
```

Comment:
kenrel -> kernel and rename the directories appropriately

leave the / after MAGMA_ROOT just for readability